webapp/src/DataFixtures/GatlingDataFixture.php

```diff
@@ -0,0 +1,129 @@
+<?php declare(strict_types=1);
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Bundle\FixturesBundle\FixtureGroupInterface;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+
+use App\Entity\Contest;
+use App\Entity\ContestProblem;
+use App\Entity\Language;
+use App\Entity\Team;
+use App\Entity\TeamCategory;
+use App\Entity\Role;
+use App\Entity\User;
+
+/**
+ * Class GatlingDataFixture
+ * @package App\DataFixtures
+ */
+class GatlingDataFixture extends Fixture implements FixtureGroupInterface, DependentFixtureInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public static function getGroups(): array
+    {
+        return ['gatling'];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function load(ObjectManager $manager)
+    {
+        $num_jury_accounts = 25;
```

Comment:
no team accounts?

Reply:
Nope, it uses self registration to create and act as team accounts. It's be trivial to change it to use pre-created accounts, but this let's it stress a bit more of the system.

Reply:
SGTM